pkg/resource/image_metadata.go

```diff
@@ -43,6 +43,15 @@ const (
 	// LabelKeyOrgKieProtoBuf is the label key for ProtoBuf metadata
 	LabelKeyOrgKieProtoBuf = "org.kie" + labelNamespaceSep + "persistence" + labelNamespaceSep + "proto"
 
+	// LabelPrometheusScrape is the label key for prometheus scrape configuration
+	LabelPrometheusScrape = LabelKeyPrometheus + "/scrape"
```

Comment:
I remember setting those constants somewhere else. Is they new?

Reply:
They are used in some unit tests. I have modified the unit tests to use the const variables.